Demos/04-add-msgraph/app/build.gradle

```diff
@@ -16,6 +16,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    packagingOptions{
+        pickFirst 'META-INF/jersey-module-version'
```

Comment:
I didn't see this in the module 4 README, but I had to add this line to get the project to run after doing the final steps in the README. Should this get added?

Reply:
Yes, added it.